interface/patient_tracker/patient_tracker.php

```diff
@@ -86,7 +86,7 @@
 $datetime = date("Y-m-d H:i:s");
 # go get the information and process it
 $appointments = fetch_Patient_Tracker_Events($from_date, $to_date, $provider, $facility, $form_apptstatus, $form_apptcat);
-$appointments = sortAppointments( $appointments, 'date','time' );
+$appointments = sortAppointments( $appointments,'date','time' );  // TODO - TIME IS NOT A PARAMETER for this function
```

Comment:
The time sort is needed, but the function call only support 2 arguments, time is not one of them...